backend/pokemon/tests.py

```diff
@@ -1,4 +1,23 @@
-# from django.test import TestCase
+from django.test import TestCase
 
+from pokemon.helpers import get_pokemon_from_api, is_team_valid
 
-# Create your tests here.
+
+class PokemonSumTest(TestCase):
+    def test_sum_success(self):
+        pokemon = [
+            get_pokemon_from_api("pidgeot"),
```

Comment:
Let's not test getting the pokemon from the API. It's not a good practice to allow tests to make requests for external APIs. Please get the pokemons from the database